.github/workflows/docusaurus.yml

```diff
@@ -19,6 +19,22 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.64.0
+          override: true
+
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --features vendored --manifest-path compiler/Cargo.toml --bin relay-config-schema --release
+      
+      - name: Generate JSON Schema
+        run: |
+          ./compiler/target/release/relay-config-schema > website/static/relay-config-schema.json
```

Comment:
We are doing this so people can reference the JSON schema via the Relay website?
In my old PR I opted into including the JSON schema in the relay-compiler npm package, so that the JSON schema would match the installed version of the compiler in the future.

Reply:
This makes way more sense and would be versioned appropriately. I'll make this change tonight.